drivers/video/ov5640.c

```diff
@@ -131,6 +133,7 @@ struct ov5640_mipi_frmrate_config {
 struct ov5640_mode_config {
 	uint16_t width;
 	uint16_t height;
+	uint16_t array_size_res_params;
```

Comment:
While seen in context (`ARRAY_SIZE(res_params)`) this makes sense, but here, `res_params_size` or `_len` or `_num` would be more intuitive. Not blocking at all!